scripts/fix-modules.js

```diff
@@ -0,0 +1,18 @@
+const fs = require("fs");
```

Comment:
You should prettify it. For example, we don't use double quotes in js files

Reply:
That true for the entire repo though, so we probably wanna fix it separately.
In any case, I applied it to this file only.
First, by just replacing double quotes with single quotes, which forced me to use backslashes for a couple of single quotes **within** one of the strings.
Then, by just running the prettifier on this file.
Note that in this case, the single quotes inside the string have caused the prettifier to violate its own rules, by overriding my backslash fix inside the string, and using double quotes for the string itself.
So the file now has an equally bad violation of our coding-convention.
we might be able to workaround this with ticks or some other JS syntax.